mautic/mautic
# Fix cancel scheduled campaign event log

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | x
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | Fixes #8867
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

I noticed schedule campaign  actions cannot cancel via timeline. This PR should fixed it.

![image](https://user-images.githubusercontent.com/462477/68465499-e69df580-0212-11ea-9578-080069df9144.png)


[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Create campaign
2. Setup scheduled event (Change contacts tags for example)
3. Trigger campaign
4. Go to contact profile and see schedule campaign event action
5. Try cancel it
6. Reload page and see schedule event action still exists

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Repeat all steps and see If scheduled action is non scheduled now

You should see in timeline 

![image](https://user-images.githubusercontent.com/462477/87299025-8f6fc380-c50b-11ea-8e11-4a410f7548ff.png)
